prow/config/calico-config.yaml

```diff
@@ -0,0 +1,27 @@
+# This section includes base Calico installation configuration.
+# For more information, see: https://docs.tigera.io/calico/latest/reference/installation/api#operator.tigera.io/v1.Installation
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+spec:
+  # Configures Calico networking.
+  calicoNetwork:
+    # Note: The ipPools section cannot be modified post-install.
+    ipPools:
+    - blockSize: 26
+      cidr: 10.244.0.0/16
+      encapsulation: VXLANCrossSubnet
+      natOutgoing: Enabled
+      nodeSelector: all()
+
+---
+
+# This section configures the Calico API server.
+# For more information, see: https://docs.tigera.io/calico/latest/reference/installation/api#operator.tigera.io/v1.APIServer
+apiVersion: operator.tigera.io/v1
+kind: APIServer
+metadata:
+  name: default
+spec: {}
+
```

Comment:
I cannot comment on calico-install.yaml since its too big, so commenting here.

We cannot depend on quay.io, they have constant outages. We can mirror images to gcr.io/istio-testing

We should mention where the file is from, with steps to update it as needed

Is there a more minimal install without CRDs, and other bloat? If so can we use it? Kind of ties into my operator comment